override/merge.go

```diff
@@ -179,7 +180,7 @@ func convertIntoSequence(value any) []any {
 			i++
 		}
 		slices.SortFunc(seq, func(a, b any) int {
-			return compare(a.(string), b.(string))
```

Comment:
So, I am not sure, should we use `compare` or `cmp.Compare`? @ndeloof

Reply:
`cmp.Compare` should be safe (?), at least we already use it on go1.21+, so better just remove the compare/compare120 hack